.circleci/continue_config.yml

```diff
@@ -332,10 +332,10 @@ workflows:
                 - bigquery
                 - clickhouse-cloud
                 - athena
-          filters:
-           branches:
-             only:
-               - main
+          #filters:
+          # branches:
+          #   only:
+          #     - main
```

Comment:
Yes, I needed to show that all the tests were still running / passing (if there was an issue with these, it would only show up post-merge)

I'll uncomment this prior to merge